mne/preprocessing/ica.py

```diff
@@ -120,7 +120,23 @@ class ICA(ContainsMixin):
     u"""M/EEG signal decomposition using Independent Component Analysis (ICA).
 
     This object can be used to estimate ICA components and then remove some
-    from Raw or Epochs for data exploration or artifact correction.
+    from Raw, Epochs or Evoked for data exploration or artifact correction.
+    
+    Prior to fitting and applying the ICA, data is whitened (de-correlated and
+    scaled to unit variance, also called sphering transformation) by means of
+    a Principle Component Analysis (PCA). This step additionally introduces the
+    option to reduce the dimensionality of the data prior to fitting/reverting
+    the ICA,controllable by the two parameters `max_pca_components` (for 
+    decomposition) and `n_pca_components` (for re-composition).
+    .. note:: Commonly used for reasons of i) computational efficiency and
+              ii) additional noise reduction, it is a matter of current debate
+              whether pre-ICA dimensionality reduction could decrease reliability/
+              stability of ICA, especially during preprocessing (see e.g. [5]).
```

Comment:
change to `... decrease the reliability ... [5]`

(add "the", remove "/stability", remove (see e.g. ...) and just reference directly)

Reply:
Thanks @sappelhoff for all the suggestions. I was trying to incorporate them. I left both reliability and stability since the paper describes (at least) two different issues with dim-reduction, one being test-retest-stability and the other being inter-subject consistency of components and number of clearly localizable dipolar components (the latter I summarize by "reliability").